client/verta/verta/_internal_utils/_artifact_utils.py

```diff
@@ -200,6 +202,16 @@ def ensure_bytestream(obj):
             bytestream.seek(0)
             return bytestream, "cloudpickle"
 
+        torch = maybe_dependency("torch")
+        if torch is not None:
+            try:
+                torch.save(obj, bytestream)
+            except:  # not something torch can serialize
+                pass
+            else:
+                bytestream.seek(0)
+                return bytestream, "torch"
+
```

Comment:
I don't quite understand the logic of this part... This should be for the case where user pass a torch model object (not deserialized file) to log_model, correct? I think that case is already covered in the cloudpickling step above though, so this might not be reached at all 🤔I think that's how our existing test passed? 

Anyway, I could be very wrong; you should double check by adding a print statement here, then passed a torch model to log_model and see if this code block is reached. Running one of the existing tests that I mentioned above should suffice 🤔

Reply:
For your reference, the tests that I was talking about:
https://github.com/VertaAI/modeldb/blob/8831bd96a9ffd587d63c892b56384320060648e7/client/verta/tests/test_artifacts.py#L277
https://github.com/VertaAI/modeldb/blob/9e9a0589b2fc8a374119e7d10c97ba09bb70da28/client/verta/tests/test_endpoint/test_endpoint.py#L499

Reply:
@evgeniilezhnin 
Because the Client actually uploads using cloudpickle anyway (see thread in #client), let's remove this.
Let me know if it causes a problem.

Reply:
removed